frontend/routes/package/(_components)/Docs.tsx

```diff
@@ -83,9 +87,8 @@ export function DocsView(
             </div>
           )}
         </div>
-
         {docs.toc && (
-          <div class="max-lg:hidden col-span-1 top-0 lg:sticky lg:max-h-screen box-border space-y-4 -mt-4 pt-4">
+          <div class="max-lg:row-start-1 lg:col-[span_1_/_-1] lg:top-0 lg:sticky lg:max-h-screen box-border space-y-4 -mt-4 pt-4">
```

Comment:
FWIW, `col-span-full` can be used instead of that JIT class

Reply:
nope, that would be `1 / -1`, not `span 1 / -1`.
I could switch it to `col-start-4`, but find that a bit less clear since it doesnt state how many cols it spans for